### PR TITLE
Begins updating Locksmith with more idiomatic TS

### PR DESCRIPTION
**Pull request by akeem**
# Description

Locksmith's beginnings as a JS application has lead to a few places where we
could update the code to benefit from some of TS functionality.

This is a bit of a piecemeal effort trending in a better direction.

# Issues

Fixes #
Refs #

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [ ] This PR only contains configuration changes (package.json, etc.)
  - [x] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: locksmith/src/controllers/metadataController.ts [hunk 1]
```diff
@@ -2,16 +2,15 @@
 /* eslint-disable no-use-before-define */
 import { Response } from 'express-serve-static-core' // eslint-disable-line no-unused-vars, import/no-unresolved
 import Normalizer from '../utils/normalizer'
-import { SignedRequest } from '../types' // eslint-disable-line no-unused-vars, import/no-unresolved
```
Comment (cnasc) [inline locksmith/src/controllers/metadataController.ts]: Any reason to remove this typedef?

File: locksmith/src/routes/metadata.ts [hunk 2]
```diff
@@ -1,8 +1,8 @@
 import express from 'express'
 import signatureValidationMiddleware from '../middlewares/signatureValidationMiddleware'
+import MetaDataController from '../controllers/metadataController'
```
Comment (cnasc) [inline locksmith/src/routes/metadata.ts]:
```suggestion
import MetadataController from '../controllers/metadataController'
```

Not critical, just a naming suggestion to have it match the namespace definition and be more like the file name.

File: locksmith/src/controllers/metadataController.ts (hunk 1 above)
Comment (akeem) [reply locksmith/src/controllers/metadataController.ts]: Cutting over importing the controller in the router revealed a type mismatch that previously wasn't raised. While the types in this controller are as expected, the path mapping code in the router expects an object that contains a `Request` making this incompatible

File: locksmith/src/routes/metadata.ts (hunk 2 above)
Comment (akeem) [reply locksmith/src/routes/metadata.ts]: sounds good, i'll make this change and adjust its usage below

**Pull request merged**